packages/arb-node-core/cmdhelp/wallet.go

```diff
@@ -117,15 +118,15 @@ func GetKeystore(
 		}
 	} else if len(walletConfig.Local.PrivateKey) != 0 {
 		if walletConfig.Local.OnlyCreateKey {
-			return nil, nil, errors.New("wallet key already exists, remove --wallet.local.only-create-key to run normally")
+			message = "wallet key already exists, remove --wallet.local.only-create-key to run normally"
```

Comment:
Doesn't this change the behavior? Doesn't seem like we won't exit anymore in this case

Reply:
made things clearer